engine/table/src/main/java/io/deephaven/engine/util/CacheDir.java

```diff
@@ -0,0 +1,32 @@
+package io.deephaven.engine.util;
+
+import java.nio.file.Path;
+
+public final class CacheDir {
+    private static final String DEEPHAVEN_CACHE_DIR = "deephaven.cache.dir";
+    private static final String JAVA_IO_TMPDIR = "java.io.tmpdir";
+
+    private static final Path cacheDir;
+
+    static {
+        final String deephavenCacheDir = System.getProperty(DEEPHAVEN_CACHE_DIR);
```

Comment:
`Configuration.getInstance().getPropertyWithDefault`?

Reply:
^ this is not an existing method. But I can use Configuration, sure.

Reply:
Sorry, method is `io.deephaven.configuration.PropertyFile#getStringWithDefault`.